README.md

```diff
@@ -710,6 +710,10 @@ Example:
 var socket = io.connect();
 var socketNamespaced = io.connect('/rnd');
 
+socket.on('randomUpdate',function(data){
```

Comment:
there are lint issues throughout the readme and the examples.

Reply:
i'll run the lint fixer on them

Reply:
how do you identify lint issues in readme?

Reply:
you mean actually with your eyes?? or run eslint on it somehow?

Reply:
yes, actually with my eyes.

Reply:
ok :) i'll try, not  sure i know all the lint rules exactly. are we  talking about spaces in the right places etc..?

Reply:
:) yes. but as an alternative, you can c/p the examples into a source file and run the linter.

Reply:
on it

Reply:
ok i've gone over entire readme. fixed a few lint issues...